src/test/java/org/opendcs/authentication/UserAuthFileTest.java

```diff
@@ -19,6 +19,7 @@ public class UserAuthFileTest
     @Test
     public void test_user_auth_file_default(@TempDir Path configDir) throws Exception
     {
+        System.out.println(configDir.resolve("test.txt").toString());
```

Comment:
debug? If not, should have a little more context output.

Reply:
Oops, that was supposed to be removed. Was trying to get some initial feedback as to what exactly was going on.